tests/unit/CategoryUtilsTest.ts

```diff
@@ -19,3 +20,85 @@ describe(`isMissingCategory`, () => {
         expect(isCategoryMissing('Meals')).toBe(false);
     });
 });
+
+describe('isCategoryDescriptionRequired', () => {
+    const mockPolicyCategories: PolicyCategories = {
+        Travel: {
+            areCommentsRequired: true,
+            enabled: true,
+            name: 'Travel',
+            pendingAction: null,
+        },
+        Meals: {
+            areCommentsRequired: false,
+            enabled: true,
+            name: 'Meals',
+            pendingAction: null,
+        },
+        Office: {
+            areCommentsRequired: true,
+            enabled: false,
+            name: 'Office',
+            pendingAction: null,
+        },
+    };
+
+    it('returns false when policyCategories is undefined', () => {
+        expect(isCategoryDescriptionRequired(undefined, 'Travel', true)).toBe(false);
+    });
+
+    it('returns false when category is undefined', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, undefined, true)).toBe(false);
+    });
+
+    it('returns false when category is empty string', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, '', true)).toBe(false);
+    });
+
+    it('returns false when areRulesEnabled is undefined', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, 'Travel', undefined)).toBe(false);
+    });
+
+    it('returns false when areRulesEnabled is false', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, 'Travel', false)).toBe(false);
+    });
+
+    it('returns true when category has areCommentsRequired set to true and rules are enabled', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, 'Travel', true)).toBe(true);
+    });
+
+    it('returns false when category has areCommentsRequired set to false even if rules are enabled', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, 'Meals', true)).toBe(false);
+    });
+
+    it('returns true when category has areCommentsRequired set to true regardless of enabled status', () => {
+        expect(isCategoryDescriptionRequired(mockPolicyCategories, 'Office', true)).toBe(true);
+    });
```

Comment:
NAB: The name of this test makes it sound like `areRulesEnabled` should be false, but it's true.